### PR TITLE
[doc] Add doc for adding partitions

### PR DESCRIPTION
**Pull request by xuzifu666**
### Purpose

Linked issue: close #xxx

add doc for add partition.

### CODE REVIEW

**Comment by wwj6591812** [comment]
+1

File: docs/content/flink/sql-alter.md [hunk 1]
```diff
@@ -114,6 +114,21 @@ ALTER TABLE my_table DROP PARTITION (`id` = 1), PARTITION (`id` = 2);
 
 ```
 
+## Adding Partitions
+
+The following SQL adds the partitions of the paimon table.
+
+For flink sql, you can specify the partial columns of partition columns, and you can also specify multiple partition values at the same time.
```
Comment (JingsongLi) [inline docs/content/flink/sql-alter.md:121]: Note here, only with metastore configured

File: docs/content/spark/sql-alter.md [hunk 2]
```diff
@@ -119,6 +119,14 @@ The following SQL drops the partitions of the paimon table. For spark sql, you n
 ALTER TABLE my_table DROP PARTITION (`id` = 1, `name` = 'paimon');
 ```
 
+## Adding Partitions
+
+The following SQL adds the partitions of the paimon table. For spark sql, you need to specify all the partition columns.
```
Comment (JingsongLi) [inline docs/content/spark/sql-alter.md:124]: Note here, only with metastore configured

**Review by JingsongLi — APPROVED** [review]
+1

**Pull request merged**